### ISSUE TITLE
Create command should not create configuration in case of failure

### ISSUE THREAD

**Issue by chrismatix** [airyhq/airy#1601]
**Current**

The first thing the airy create command does is create the workspace configuration. If there is a subsequent failure during the provisioning and installation.

**Desired**

The configuration workspace is only created after all create steps are completed.

**Comment by paulodiniz** [comment]
I agree with this feature, but doesn't this make the `init-only` option an edge case?

Because the current behaviour if this flag is true, is that we create the config directory and exit the program. If we move `creating the config dir` as the *last* part of the process it almost seems like this action needs it's own command

What do you think?

**Comment by chrismatix** [comment]
@paulodiniz There was a whole discussion about getting rid of `airy init` and I would not like to bring it back. So yes. `init-only` is a special case of the `create` command, because it's a part of its flow.

To keep the code simple you could also `defer` the creation of the workspace and `return` if the init only flag is set.

**Comment by paulodiniz** [comment]
@chrismatix I don't think we can do that way because if we defer the function, the other operations will run and that's the opposite of `init-only`

**Comment by paulodiniz** [comment]
To be clear on where I stand: I would remove the `init-only` option

**Comment by lucapette** [comment]
As I understand it, now the init-only option makes no sense so I agree. Let's remove it (the fact it's not mentioned anywhere in the docs is also a sign) for now

**Comment by paulodiniz** [comment]
Also, we do the `provision` of the provider using the config directory. Maybe I'm missing some context, but how can we do that if we create the config dir last?
